Helpful error message if import path doesn't exist

Currently, if you try to import a directory that doesn't exist RITA spits out a panic and trace message like the following:

```
rita import /does/not/exist/vsagent vsagent
[+] Importing /does/not/exist/vsagent
	[-] Finding files to parse
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/activecm/rita/parser.(*FSImporter).Run(0xc0000e89c0, 0xb15940, 0xc00024e850)
	/go/src/github.com/activecm/rita/parser/fsimporter.go:85 +0xe0c
github.com/activecm/rita/commands.doImport(0xc000116f20, 0x0, 0x0)
	/go/src/github.com/activecm/rita/commands/import.go:68 +0x38b
github.com/activecm/rita/commands.init.2.func1(0xc000116f20, 0x0, 0x0)
	/go/src/github.com/activecm/rita/commands/import.go:28 +0x2f
github.com/activecm/rita/vendor/github.com/urfave/cli.HandleAction(0x98e020, 0xa95b50, 0xc000116f20, 0xc000034800, 0x0)
	/go/src/github.com/activecm/rita/vendor/github.com/urfave/cli/app.go:490 +0xc8
github.com/activecm/rita/vendor/github.com/urfave/cli.Command.Run(0xa6cb20, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0xa8237f, 0x26, 0xa8f788, ...)
	/go/src/github.com/activecm/rita/vendor/github.com/urfave/cli/command.go:210 +0x990
github.com/activecm/rita/vendor/github.com/urfave/cli.(*App).Run(0xc0000b9a00, 0xc00001e0c0, 0x4, 0x4, 0x0, 0x0)
	/go/src/github.com/activecm/rita/vendor/github.com/urfave/cli/app.go:255 +0x687
main.main()
	/go/src/github.com/activecm/rita/rita.go:29 +0x129
```

Rita should instead detect if the directory exists and print a nice error message to the user rather than the cryptic error above.

Related:
- #327 
- #346

--- a/commands/import.go
+++ b/commands/import.go
@@ -2,6 +2,7 @@ package commands
 
 import (
 	"fmt"
+	"os"
 	"path/filepath"
 
 	"github.com/activecm/rita/parser"
@@ -111,11 +112,19 @@ func (i *Importer) setTargetDatabase() error {
 }
 
 func (i *Importer) setImportDirectory() error {
-	var err error
-	i.importDir, err = filepath.Abs(i.importDir)
+
+	// parse directory path
+	filePath, err := filepath.Abs(i.importDir)
 	if err != nil {
 		return cli.NewExitError(err.Error(), -1)
 	}
+
+	// check if directory exists
+	if _, err := os.Stat(filePath); os.IsNotExist(err) {
+		return cli.NewExitError(err.Error(), -1)
+	}
+
+	// assign import directory
 	i.res.Config.S.Bro.ImportDirectory = i.importDir
 	return nil
 }

--- a/database/meta.go
+++ b/database/meta.go
@@ -44,6 +44,9 @@ type (
 		Analyzed       bool          `bson:"analyzed"`        // Has this database been analyzed
 		ImportVersion  string        `bson:"import_version"`  // Rita version at import
 		AnalyzeVersion string        `bson:"analyze_version"` // Rita version at analyze
+		Rolling        bool          `bson:"rolling"`
+		TotalChunks    int           `bson:"total_chunks"`
+		CurrentChunk   int           `bson:"current_chunk"`
 		TsRange        Range         `bson:"ts_range"`
 	}
 )

